src/component/ui/cards/ArticleCard2.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Glow from '../buttons/glow/Glow.jsx';
+import './Cards.scss';
+import Rating from '../Rating/Rating';
+
+const ArticleCard2 = (props) => {
+  const {
+    ArticleTitle,
+    Article,
+    ArticleImage,
+    userImage,
+    username,
+    updatedTime,
+  } = props;
+  return (
+    <div className="article-Card-2" style={{ border: '0rem' }}>
+      <h5 className="card-title">{ArticleTitle}</h5>
+      <img className="card-img-top" src={ArticleImage} alt="Card" />
+      <div className="card-body" style={{ padding: '0rem' }}>
+        <br />
+        <p className="card-text">{Article}</p>
+        <span><img className="user-image" src={userImage} alt="user" /></span>
+        <p className="col-8 col-md-8 user-name" style={{ display: 'inline-block' }}>{username}</p>
```

Comment:
Hi @akpante3 I think using style directly here defeats the idea of using className and having separate scss file.

Reply:
thats true .. will make the change